src/__main__.py

```diff
@@ -16,14 +16,12 @@ def __init__(self):
         self.load_extension("exts.ping")
         self.load_extension("exts.utils")
 
-
     async def on_ready(self):
         print("SUS")
 
 
-from os import getenv
-
 bot = Impostor()
-
-if getenv("PR_TESTING") in ["0", None]:
+if not config.pr_testing:
     bot.run(getenv("TOKEN"))
+else:
+    print("More SUS")
```

Comment:
what exactly is the reason of that???

Reply:
so when someone runes it when pr-mode (for some reason), they know that it actually ran. Prob should change to more descriptive.

Reply:
i mean... pr mode is just for ci checks...

Reply:
yeah... so remove?